python_apps/pypo/pypo/pypofetch.py

```diff
@@ -336,15 +336,15 @@ def process_schedule(self, schedule_data):
             download_dir = self.cache_dir
             try:
                 os.makedirs(download_dir)
-            except Exception, e:
+            except Exception as e:
                 pass
 
             media_copy = {}
             for key in media:
                 media_item = media[key]
                 if (media_item['type'] == 'file'):
                     fileExt = self.sanity_check_media_item(media_item)
-                    dst = os.path.join(download_dir, unicode(media_item['id']) + unicode(fileExt))
+                    dst = os.path.join(download_dir, str(media_item['id']) + str(fileExt))
```

Comment:
No need to convert string to string:

```suggestion
                    dst = os.path.join(download_dir, media_item['id'])+ fileExt)
```